bangazonapi/views/lineitem.py

```diff
@@ -49,7 +49,7 @@ def retrieve(self, request, pk=None):
             HTTP/1.1 204 No Content
         """
         try:
-            # line_item = OrderProduct.objects.get(pk=pk)
+            line_item = OrderProduct.objects.get(pk=pk)
```

Comment:
This line of code should be deleted. The query below on line 54 gets the same data from the database with an added layer of filtering to ensure we only get back data that belongs to the authenticated user.

Reply:
Ok!! Good to know!!